manifests/mod/negotiation.pp

```diff
@@ -11,10 +11,13 @@
 #
 class apache::mod::negotiation (
   Variant[Array[String], String] $force_language_priority = 'Prefer Fallback',
-  Variant[Array[String], String] $language_priority = ['en', 'ca', 'cs', 'da', 'de', 'el', 'eo', 'es', 'et',
-    'fr', 'he', 'hr', 'it', 'ja', 'ko', 'ltz', 'nl', 'nn',
-    'no', 'pl', 'pt', 'pt-BR', 'ru', 'sv', 'zh-CN',
-  'zh-TW'],
+  Variant[Array[String], String] $language_priority      =
+    [
+      'en', 'ca', 'cs', 'da', 'de', 'el', 'eo', 'es', 'et',
+      'fr', 'he', 'hr', 'it', 'ja', 'ko', 'ltz', 'nl', 'nn',
+      'no', 'pl', 'pt', 'pt-BR', 'ru', 'sv', 'zh-CN',
+      'zh-TW'
+    ],
```

Comment:
Linting fails on this which breaks CI.

Reply:
I restored the initial Version from `main`.